src/session/user.rs

```diff
@@ -302,6 +341,13 @@ impl User {
         )
         .upcast()
     }
+    pub fn deleted_name_expression(user_expression: &gtk::Expression) -> gtk::Expression {
```

Comment:
... I suggest deleting this and creating a `User::display_name_expression()`, that it does the following:

```rust
let user_type_expression = ...
let user_full_name_expression = ...
// A closure expression where it returns the display name based on the user type ('Deleted Account' in case of a deleted user, otherwise the full name)
```